report/README.txt

```diff
@@ -0,0 +1 @@
+// This directory is intended to hold the junit XML reports generated by fv and unit tests.
```

Comment:
Do we really need to check this in?

I'd suggest we make the directory if it doesn't exist, and add it to `.gitignore`.

Reply:
This is what we've done in other repos (node, felix, etc), but OK.  

Where would be the right place to make that directory?  Its referred to in the test code, so would I create it in there?   Or in the makefile?  Having the directory checked in at least means that even if people run the tests without using the makefile (i.e. directly running `ginkgo` or `fv.test` the directory will be there).

Reply:
If we've done it elsewhere, then let's be consistent at least. SGTM